packages/core/entity-detail/src/components/DetailForm/DetailForm.js

```diff
@@ -52,16 +53,21 @@ const DetailForm = props => {
     [submitting, mode, valid, anyTouched, formErrors, intl]
   )
 
+  const modifiedFormDefinition = useMemo(
+    () => (formDefinition ? modifyFormDefinition(formDefinition) : formDefinition),
+    [formDefinition, modifyFormDefinition]
+  )
+
   const embedType = env.getEmbedType()
 
   return (
     <StyledForm {...formEventProps} ref={formEl}>
       <form.FormBuilder
         entity={entity}
         formName={formName}
-        formDefinition={formDefinition}
+        formDefinition={modifiedFormDefinition}
```

Comment:
Here we have the same situation as we have for the `entity-list`. The `formDefinition` is also used in `DetailView` for the validation...